internal/ingress/controller/parser/kongstate/consumer_test.go

```diff
@@ -46,6 +47,24 @@ func TestConsumer_SetCredential(t *testing.T) {
 			},
 			wantErr: false,
 		},
+		{
+			name: "key-auth without key",
+			args: args{
+				credType: "key-auth",
+				consumer: &Consumer{
+					Consumer: kong.Consumer{
+						Username: &username,
+					},
+				},
```

Comment:
```
emptyConsumer := Consumer{
	Consumer: kong.Consumer{
		Username: &username,
	},
}
```

this way we could replace `consumer: ...` with `consumer: &emptyConsumer` (and the same for `result`, saving all the repetitions.

Reply:
Left independent but squashed into a single line per literal as discussed.

Reply:
As per discussion elsewhere: we remain with literals in every test case but roll them up in a single line.